airflow/www/auth.py

```diff
@@ -28,6 +28,10 @@
 T = TypeVar("T", bound=Callable)
 
 
+def get_access_denied_message():
+    return conf.get("webserver", "access_denied_message")
```

Comment:
This should either use `get_mandatory` or provide a fallback.

Reply:
Not really. The `get_mandatory` is needed only when we want to have a value but the value has no default (for example when we want to get a URL that has no default but we **really** need it to be configured). 

The way it works when it is put in config.yml is that "default" is automatically used as fallback if there is no fallback (It's been working like that already for quite some time via defaul_config.cfg  - so my recent change has not changed it - jut made it works without the default_config.cfg

Reply:
I don't think it is needed as @potiuk mentions here. The older default text is passed as the default in the new property in config.yml, which is basically the fallback.